### ISSUE TITLE
Proposal: Comments

### ISSUE THREAD

**Issue by lyxal** [bigyihsuan/International-Phonetic-Esoteric-Language#4]
Some sort of a commenting system would be nice:

I reckon the command should either be `#` or some other IPA symbol of your choosing.

For example:
```
1 #This will be the loop counter
91t #Push 10 onto the stack
œ #Start a loop ten times
ɟo #Print the counter
1t #Increment by 1
ɶ #Close loop

#191tœɟo1tɶ
```

**Comment by bigyihsuan** [comment]
Closed with commit d76cc49.

**Comment by bigyihsuan** [comment]
This has been in the works, I decided to use parentheses for comments, similar to Forth.
